discovery/src/main/java/com/ibm/watson/discovery/v2/model/ComponentSettingsAggregation.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.discovery.v2.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/**
+ * Display settings for aggregations.
+ */
+public class ComponentSettingsAggregation extends GenericModel {
+
+  /**
+   * Type of visualization to use when rendering the aggregation.
+   */
+  public interface VisualizationType {
+    /** auto. */
+    String AUTO = "auto";
+    /** facet_table. */
+    String FACET_TABLE = "facet_table";
+    /** word_cloud. */
+    String WORD_CLOUD = "word_cloud";
+    /** map. */
+    String MAP = "map";
+  }
+
+  private String name;
+  private String label;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)